### PR TITLE
add NetworkedMultiplayerENet::set_bind_ip

### PR DESCRIPTION
**Pull request by Marqin**
which allows ENet to bind on custom IP.
#6257 but rebased

### CODE REVIEW

**Comment by akien-mga** [comment]
Thanks :)

File: modules/enet/networked_multiplayer_enet.cpp [hunk 1]
```diff
@@ -610,12 +610,12 @@ void NetworkedMultiplayerENet::_bind_methods() {
 	ObjectTypeDB::bind_method(_MD("close_connection"),&NetworkedMultiplayerENet::close_connection);
 	ObjectTypeDB::bind_method(_MD("set_compression_mode","mode"),&NetworkedMultiplayerENet::set_compression_mode);
 	ObjectTypeDB::bind_method(_MD("get_compression_mode"),&NetworkedMultiplayerENet::get_compression_mode);
+	ObjectTypeDB::bind_method(_MD("set_bind_ip", "ip"),&NetworkedMultiplayerENet::set_bind_ip);
```
Comment (akien-mga) [inline modules/enet/networked_multiplayer_enet.cpp]: Shouldn't there be a corresponding getter?

**Comment by akien-mga** [comment]
Code and feature look fine.

File: modules/enet/networked_multiplayer_enet.cpp (hunk 1 above)
Comment (Marqin) [inline modules/enet/networked_multiplayer_enet.cpp]: I didn't thought about that.

**Pull request merged**